front/pages/manage_participants.js

```diff
@@ -0,0 +1,132 @@
+/*
+ * Manage the participants of event/citizen action
+ */
+export default() => {
+    const allCheckboxes = dom('#members-check-all');
+    const selectionCheckboxes = document.getElementsByName('members[]');
+    const contactButton = dom('#members-contact-button');
+    const contactSelection = dom('#members-contact-selection');
+    const exportButton = dom('#members-export-button');
+    const exportSelection = dom('#members-export-selection');
+    const printButton = dom('#members-print-button');
+    const printSelection = dom('#members-print-selection');
+
+    // Toggle action buttons
+    function toggleButtons(value) {
+        if (null != contactButton) contactButton.disabled = !value;
+        if (null != exportButton) exportButton.disabled = !value;
+        if (null != printButton) printButton.disabled = !value;
+    }
+
+    // Toggle each 'selection' checkbox
+    function toggleCheckboxs(value) {
+        const selectCheckboxes = (element) => {
+            element.checked = !!value;
+        };
+
+        selectionCheckboxes.forEach(selectCheckboxes);
```

Comment:
can you try to avoid the constant declaration? 
I'm wondering for a visibility of `value` variable in this case :)
```js
selectionCheckboxes.forEach((element) => {
  element.checked = !!value;
});
```